src/models/geodetic.py

```diff
@@ -322,8 +322,9 @@ def point2sources(self, point):
                 tpoint.pop(hyper)
 
         source_params = self.sources[0].keys()
+        source_params_arr= num.array(list(tpoint.keys()))
 
```

Comment:
why array and not list?

Reply:
the iteration in the next lines will else not work, as the space to iterate over will change if using a dict